lib/fastlane_core/project.rb

```diff
@@ -114,13 +116,18 @@ def select_scheme
       if schemes.count == 1
         options[:scheme] = schemes.last
       elsif schemes.count > 1
-        if Helper.is_ci?
-          Helper.log.error "Multiple schemes found but you haven't specified one.".red
-          Helper.log.error "Since this is a CI, please pass one using the `scheme` option".red
-          raise "Multiple schemes found".red
+        preferred = schemes.find_all { |a| a.downcase.include?(preferred_to_include.downcase) }
```

Comment:
I didn't know about `casecmp`, we can use it to compare strings, thanks :+1: 

Yeah, it's used for automatically selecting the UI Tests targets when using `snapshot`, so it would search for `UITest` by default, which might be `UITestExample`